builds/jazz-build-module/whitelist-validator-module.groovy

```diff
@@ -0,0 +1,139 @@
+#!groovy?
+
+whiteListContent = '''
+resources:
```

Comment:
Let's put this into its own yml file in build-module - easy to maintain.

Reply:
Ok the whitelist file has been moved into jazz-build-module/sls-app/whitelist.yml

Reply:
Resolved with https://github.com/tmobile/jazz/pull/829/commits/70374d203a7cb8338de2d84ee1c4902831a57b35 and https://github.com/tmobile/jazz/pull/829/commits/0af8f0b94a3ac56f5d2d813795b2ce99cf092e41

Reply:
Also https://github.com/tmobile/jazz/pull/829/commits/89ef86def038b552ad9a3494670f62835689e633 is involved